canvas_manage_course/settings/base.py

```diff
@@ -194,6 +186,8 @@
 
 _DEFAULT_LOG_LEVEL = SECURE_SETTINGS.get('log_level', logging.DEBUG)
 _LOG_ROOT = SECURE_SETTINGS.get('log_root', '')  # Default to current directory
+_LOG_ROOT= ''
```

Comment:
Should this be included?

Reply:
Nope, I think it was a local override. good catch! will clean it up on develop directly